src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+$bg-color: #fff;
+$border-color: #f3f3f3;
```

Comment:
you have a variable for the border but use the button color for it?
u don't use this variable